Expensify/App
# Add the currency sign to IOU request flow

### Details


### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/164533

### Tests
Request Money

- Click the + icon and select Request Money
- Enter $50 on the IOUAmount page and click next
- Observe that the header of the IOUParticipants page reads Request $50
![image](https://user-images.githubusercontent.com/521248/118733234-3152f180-b83c-11eb-8239-c748ef3ea6a8.png)
- Select a participant to move on to the IOUConfirm page
- Observe that the header & confirmation button both read Request $50
![image](https://user-images.githubusercontent.com/521248/118733245-3617a580-b83c-11eb-9817-c6f9db5ed421.png)


Split Bill

- Click the + icon and select Split Bill
- Enter $50 on the IOUAmount page and click next
- Observe that the header of the IOUParticipants page reads Split $50
![image](https://user-images.githubusercontent.com/521248/118733091-f0f37380-b83b-11eb-8df8-ca9b35354c7f.png)
- Select a couple of participants and click next
- Observe that the header of the IOUConfirm page also reads Split $50
![image](https://user-images.githubusercontent.com/521248/118733216-2b5d1080-b83c-11eb-8503-8a6b7d613054.png)

### Tested On
